release-notes/opensearch-flow-framework.release-notes-2.12.0.0.md

```diff
@@ -0,0 +1,6 @@
+## 2024-02-20 Version 2.12.0.0
```

Comment:
Is this top line needed now? Previous formats we dont have that - ref: https://raw.githubusercontent.com/opensearch-project/anomaly-detection/main/release-notes/opensearch-anomaly-detection.release-notes-2.10.0.0.md

Can we confirm with infra team that adding this line won't break release notes distribution compilation for the release? cc @opensearch-project/infra-read-only

Reply:
Looking at Job Scheduler release notes for reference, this line should just be `## Version 2.12.0.0`

Ref : https://github.com/opensearch-project/job-scheduler/blob/main/release-notes/opensearch.job-scheduler.release-notes-2.11.0.0.md?plain=1

Reply:
I referred [custom-codec](https://github.com/opensearch-project/custom-codecs/blob/main/release-notes/opensearch-custom-codecs.release-notes-2.10.0.0.md) which is one of the latest released plugin for the template. Let me know if you think otherwise, I can push the change.

Reply:
Lgtm

Reply:
FYI, 
 - security dashboards plugin matches this format: https://github.com/opensearch-project/security-dashboards-plugin/blob/b1e986cbd5e5706182be0324550a4e1d238276d2/release-notes/opensearch-security-dashboards-plugin.release-notes-2.9.0.0.md
 - index management plugin puts the date after the version: https://github.com/opensearch-project/index-management/blob/main/release-notes/opensearch-index-management.release-notes-2.9.0.0.md

I'm not sure there's a standard.  It's kind of a guess which day the release will happen, though... and doesn't say which time zone it's in.  OK merging as is, we can always change it later.